src/data/storage.js

```diff
@@ -197,44 +231,48 @@ function updateTimeSpent(originData, originStats, recentTimeSpent, isMonetizedTi
 }
 
 /**
- * Store the path to the origin's favicon in the origin data. If a relative path
- * is provided, construct the absolute path. Attempt to fetch the favicon to check
- * if it is a valid path. If fetch response is 200 OK, the path is valid, so store
- * the url of the favicon.
+ * Makes an absolute path from an input path which is either relative or absolute. If the input path
+ * is absolute, the input path is returned unchanged. If the input path is relative, an absolute
+ * path on the given origin is made with the input path.
  *
- * @param {AkitaOriginData} originData The origin data to update.
- * @param {String} faviconData The absolute or relative path to the site's favicon.
+ * @param {String} path A path which you want as an absolute path.
```

Comment:
rewording suggestion: A path to convert to an absolute path, or check that it is already an absolute path.